src/debug/di/DI.props

```diff
@@ -77,6 +77,7 @@
     <CppCompile Include="@(SourcesShim)" />
     <CppCompile Include="@(SourcesRightside)" />
     <AssembleAmd64 Condition="'$(BuildArchitecture)' == 'amd64'" Include="..\amd64\floatconversion.asm" />
+    <AssembleArm64 Condition="'$(BuildArchitecture)' == 'arm64'" Include="..\arm64\floatconversion.asm" />
```

Comment:
This file should not be included for cross_compile cases, is there a good way to do that here ?